frontend/src/components/form/api/__tests__/ApiMock.js

```diff
@@ -0,0 +1,87 @@
+function mockPromiseResolving (value) {
+  return new Promise((resolve, reject) => {
+    const timer = setTimeout(() => {
+      clearTimeout(timer)
+      resolve(value)
+    }, 100)
```

Comment:
Warum reichen 100 Millisekunden? Was garantiert uns dass wir das nicht beim nächsten Test auf 200 Millisekunden hochschrauben müssen? Könnte man auch 0 Millisekunden nehmen? In JavaScript ist ein Timeout von 0 Millisekunden ausreichend, um auf die nächste Runde des Event Loops zu warten. Vielleicht brauchts auch gar keinen Timeout?
```js
return new Promise((resolve) => {
  resolve(value)
})
```

Reply:
Ist von ApiWrapper kopiert

Reply:
Ja, habe ich inzwischen auch gesehen. Das sollte dann wenn möglich auch dort abgelöst werden.